### ISSUE TITLE
How to get embeddings for even sized window of tokens?

### ISSUE THREAD

**Issue by bxff** [UKPLab/sentence-transformers#1704]
I am trying to get embeddings for a fixed window size, which would be the maximum input size of the model. For example it could be 128 tokens for the first embedding, and 78 tokens for the second embedding for a document of 206 tokens. 

I could do this manually using the `transformers` library, using [this example](https://www.sbert.net/examples/applications/computing-embeddings/README.html#sentence-embeddings-with-transformers), I could do the splitting manually but then I would loss some of the features of the sentence-transformers, notably the multi-gpu/cpu embedding. Is there a native way of doing this, maybe passing in the tokens manually into the embedder?

How I could go about this in a manual manor:
```python3
# ...from the example before, I could change...
encoded_input = tokenizer(
	_text,
	padding=True,
	truncation=False, 
	# max_length=128, 
	# return_tensors='pt'
)

# Something like this, but in a loop using the length of all the tokens:
encoded_input['input_ids'] = encoded_input['input_ids'][0:128]
encoded_input['token_type_ids'] = encoded_input['token_type_ids'][0:128]
encoded_input['attention_mask'] = encoded_input['attention_mask'][0:128]
```

Thanks a lot.
